static/movies/style.css

```diff
@@ -0,0 +1,194 @@
+html {
```

Comment:
I can't tell if this is exactly the same as the `static/style.css` but they look almost identical, are both necessary? I'd assume `static/style.css` has the 'core' styles and we'd only use this file if there was some small overrides, but wouldn't expect to re-style `html`, `body`, etc here